pkg/operator/ceph/object/spec.go

```diff
@@ -280,17 +280,34 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 }
 
 func (c *clusterConfig) createCaBundleUpdateInitContainer(rgwConfig *rgwConfig) v1.Container {
-	caBundleMount := v1.VolumeMount{Name: caBundleVolumeName, MountPath: caBundleSourceCustomDir, ReadOnly: true}
+	caBundleMount := v1.VolumeMount{Name: caBundleVolumeName, MountPath: "/tmp/ca-bundle", ReadOnly: true}
 	volumeMounts := append(controller.DaemonVolumeMounts(c.DataPathMap, rgwConfig.ResourceName, c.clusterSpec.DataDirHostPath), caBundleMount)
 	updatedCaBundleDir := "/tmp/new-ca-bundle/"
 	updatedBundleMount := v1.VolumeMount{Name: caBundleUpdatedVolumeName, MountPath: updatedCaBundleDir, ReadOnly: false}
 	volumeMounts = append(volumeMounts, updatedBundleMount)
 	return v1.Container{
 		Name:    "update-ca-bundle-initcontainer",
 		Command: []string{"/bin/bash", "-c"},
-		// copy all content of caBundleExtractedDir to avoid directory mount itself
 		Args: []string{
-			fmt.Sprintf("/usr/bin/update-ca-trust extract; cp -rf %s/* %s", caBundleExtractedDir, updatedCaBundleDir),
+			fmt.Sprintf(`
+				OS_ID=$(grep '^ID=' /etc/os-release | cut -d'=' -f2 | tr -d '"')
```

Comment:
Does the indenting make this indented in the pod spec? Please post what `kubectl get pod -o yaml pod rook-ceph-rgw-<etc.>` looks like so we can know if we have to handle this.

Reply:
I am not quite understanding the question, But this does it detects containers OS version and according to OS version gets ca-bundle and combines it with user provided ca-bundle.

Reply:
The script is indented by 4 tabs, and I'm not sure how golang handles that inside the multiline string. I'd like to make sure that the rendered script on the deployment/pod  doesn't have unnecessary indentation.